feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/jpa/hibernate/4/module.xml

```diff
@@ -38,8 +38,6 @@
     </resources>
 
     <dependencies>
-        <module name="org.infinispan" services="import"/>
-        <module name="org.hibernate.infinispan" services="import"/>
```

Comment:
Is there a reason this was removed? Will it work the same without these modules?

Reply:
Mostly because using the Infinispan 2lc with older versions of Hibernate shouldn't work due to WF using newer (major) versions of Infinispan.  We could revert this part of the change.

@Sanne any feedback on this?  Also see other comments about Search on this pr.